gdpr_assist/models.py

```diff
@@ -281,6 +595,9 @@ class EventLog(models.Model):
     )
     app_label = models.CharField(max_length=255)
     model_name = models.CharField(max_length=255)
+    log_time = models.DateTimeField(auto_now_add=True)
+    acting_user = models.ForeignKey(settings.AUTH_USER_MODEL, null=True)
```

Comment:
@anochens I have tried to make it an actual FK to the user model but it might prove more difficult than I thought. They made it so the event logs can be in a separate database but Django doesn't support cross databases foreign keys. So either I revert back the user to using a string representation of the user, or I remove cross database support for the logs since we won't be using that. WDYT?

Reply:
we can remove the cross database thing. we weren't going to use it anyway